utils/Constants.ts

```diff
@@ -48,28 +57,37 @@ export const SECONDARY_COLLATERAL_TOKEN_NAME: { [key: string]: string } = {
   rsk: "ETHs"
 };
 
-export const numberOfLPs = 11;
-
-export const initialBdstableMintingAmount = (networkName = "mainnetFork"): BigNumber => {
-  switch (networkName) {
-    case "rsk": {
-      return to_d18(15e3);
-    }
-    case "mainnetFork": {
-      return to_d18(30e3);
-    }
-    default: {
-      return to_d18(15e3);
-    }
-  }
-};
-
 export const INITIAL_BDEU_UNISWAP_EUR_AMOUNT = 500;
 export const INITIAL_BDUS_UNISWAP_USD_AMOUNT = 500;
+export const INITIAL_BXAU_UNISWAP_XAU_AMOUNT = 0.3;
+export const INITIAL_BGBP_UNISWAP_GBP_AMOUNT = 500;
 export const INITIAL_BDX_UNISWAP_EUR_AMOUNT = 9000;
 export const INITIAL_BDX_UNISWAP_USD_AMOUNT = 10000;
 export const INITIAL_BDX_AMOUNT_FOR_BDSTABLE = to_d18(1e5);
 
+const initalBDStablePerSymbol = {
+  rsk: {
+    XAU: to_d18(INITIAL_BXAU_UNISWAP_XAU_AMOUNT), // TODO: Define amount
+    GBP: to_d18(INITIAL_BGBP_UNISWAP_GBP_AMOUNT) // TODO: Define amount
+  },
+  mainnetFork: {
+    XAU: to_d18(10)
+  }
+};
+
+export const initialBdstableMintingAmount = (networkName = "mainnetFork", symbol?: string): BigNumber => {
+  const initalAmountPerSymbol = symbol && _.get(initalBDStablePerSymbol, [networkName, symbol]);
+  if (initalAmountPerSymbol) {
+    return initalAmountPerSymbol;
+  } else if (networkName === "rsk") {
+    return to_d18(15e3);
+  } else if (networkName === "mainnetFork") {
+    return to_d18(30e3);
+  } else {
+    return to_d18(15e3);
```

Comment:
This part is dangerous. 
IMO let's add also all the amounts of existing tokens (BDEU & BDUS) and for mainnetfork. If we didn't define them then let's throw an error instead.

Reply:
great point, totally agree.